webxrlayers-1.bs

```diff
@@ -0,0 +1,730 @@
+
+GPU layer and view creation {#gpulayer}
+===========================
+
+Overview {#xrgpulayeroverview}
+--------
+<section class="non-normative">
+
+When a layer is created it is backed by a GPU resource, typically a texture, provided by one of the Web platform's graphics APIs. In order to 
+specify which API is providing the layer's GPU resources an Layer Factory for the API in question must be created.
+Each graphics API may have unique requirements that must be satisfied before a context can be used in the creation of a layer factory. For example, 
+a {{WebGLRenderingContext}} must have its xrCompatible flag set prior to being passed to the constructor of the {{XRWebGLLayerFactory}} instance.
+
+Any interaction between the {{XRSession}} the graphics API, such as allocating or retrieving textures, will go through this {{XRWebGLLayerFactory}} instance, and the exact mechanics 
+of the interaction will typically be API specific. This allows the rest of the WebXR API to be 
+graphics API agnostic and more easily adapt to future advances in rendering techniques.
+
+Once a layer factory instance has been acquired, it can be used to create a variety of {{XRLayer}}. Any layers created by that layer factory will then be able 
+to query the associated GPU resources each frame, generally expected to be the native API's texture interface.
+
+The various layer types are created with the request____Layer series of methods on the layer factory instance. Information about the graphics resources required, 
+such as whether or not to allocate a depth buffer or alpha channel, are passed in at layer creation time and will be immutable for the lifetime of the layer.
+The method will return a promise that will resolve to the associated XRLayer type once the graphics resources have been created and the layer is ready to be displayed.
+
+Some layer types may not be supported by the {{XRSession}}. If a layer type isn't supported the returned Promise will reject. {{XRProjectionLayer}} must be supported by all {{XRSession}}s.
+</section>
+
+XRProjectionLayerInit {#xrprojectionlayerinittype}
+---------------------
+The {{XRProjectionLayerInit}} dictionary represents a set of configurable values that describe how a {{XRProjectionLayer}}
+is initialized.
+
+<pre class="idl">
+dictionary XRProjectionLayerInit {
+  boolean depth = true;
+  boolean stencil = false;
+  boolean alpha = true;
+  double scaleFactor = 1.0;
+};
+</pre>
+
+The <dfn dict-member for="XRProjectionLayerInit">depth</dfn> attribute defines if the {{XRProjectionLayer}} will have a depth buffer.
+
+The <dfn dict-member for="XRProjectionLayerInit">stencil</dfn> attribute defines if the {{XRProjectionLayer}} will have a stencil buffer.
+
+The <dfn dict-member for="XRProjectionLayerInit">alpha</dfn> attribute defines if the {{XRProjectionLayer}} will have an alpha channel.
+
+The <dfn dict-member for="XRProjectionLayerInit">scaleFactor</dfn> attribute defines the value that the |session|'s 
```

Comment:
This description is accurate for the nativeScaleFactor, as you have below, but this `scaleFactor` is only the value that the recommended resolution should be multiplied by when allocating attachments. May want to change it to something along the lines of:

> The <dfn dict-member for="XRProjectionLayerInit">scaleFactor</dfn> attribute defines the value that the |session|'s [=recommended WebGL framebuffer resolution=] must be multiplied by when determining the resolution of the layer's attachments.

Reply:
done